Adding a ESP32 based board called JRC board

JRCBoard opened the pull request:
For details about technical aspects of JRC Board, the Github link is: github.com/jrcboard
And for board update and description: www.jrcboard.com

JRCBoard committed:
Adding a ESP32 based board called JRC board 

For details about technical aspects of JRC Board, the Github link is: github.com/jrcboard
And for board update and description: www.jrcboard.com

JRCBoard committed:
Create JRCBoard_example_wokwi.md

In this file, the link of sample video of JRC Board example in Wokwi platform can be found.

urish commented:
Thanks for the pull request, I reviewed it.

Unfortunately - the SVG file is not a true vector file, rather a PNG image embedded within an SVG. We need it to be a vector file. I also updated the [guidelines](https://docs.google.com/document/d/1K59xkKL70uN_c_GNCs_Xk09F3tK12B-ygmFbw6i14t4/edit) to clarify that the graphics needs to be vector.

Also, please remove JRCBoard_example_wokwi.md from the root directory (or move it to live under the same directory of the board.svg/board.json files and rename to README.md)

JRCBoard committed:
Delete JRCBoard_example_wokwi.md

JRCBoard committed:
Delete boards/board-frontech-JRC/board.json

JRCBoard committed:
Delete boards/board-frontech-JRC/board.svg

JRCBoard committed:
Add files via upload

JRCBoard committed:
Delete boards/JRC board/JRC board directory

JRCBoard committed:
Add files via upload

JSON and SVG file of JRC Board (Version: 1.0). More details on https://www.jrcboard.com

JRCBoard committed:
Merge branch 'wokwi:main' into main

JRCBoard commented:
I have Uploaded the high resolution SCG File of JRC Board (Version-1) which is a true vector file.
 Also, we have removed the requested file (JRCBoard_example_wokwi.md)
Please review it and if there is no issue, please approve it. Thanks.
